Formula/dev-nginx.rb

```diff
@@ -0,0 +1,16 @@
+class DevNginx < Formula
+  desc "Tools to configure a local development nginx to proxy our applications and services"
+  homepage "https://github.com/guardian/dev-nginx"
+  version "0.0.1"
+  url "https://github.com/guardian/dev-nginx/archive/v0.0.1.tar.gz"
```

Comment:
I've cheated and created tag 0.0.1 from https://github.com/guardian/dev-nginx/pull/5 rather than master...